### ISSUE TITLE
[Accessibility] Make boundaries of Chat visible on High Contrast mode

### ISSUE THREAD

**Issue by iwasn** [microsoft/BotFramework-WebChat#2821]
## Screenshots

![image](https://user-images.githubusercontent.com/54422491/72480966-d7f13300-37ad-11ea-979a-decca4996cbd.png)

## Version

4.5.2

> To determine what version of Web Chat you are running, open your browser's development tools, and paste the following line of code into the console.
>
> `[].map.call(document.head.querySelectorAll('meta[name^="botframework-"]'), function (meta) { return meta.outerHTML; }).join('\n')`
>
> If you are using Web Chat outside of a browser, please specify your hosting environment. For example, React Native on iOS, Cordova on Android, SharePoint, PowerApps, etc.

## Describe the bug

Boundaries of Chat should be visible in high contrast mode so that the use of this accessibility component is not disrupted.
 
## Steps to reproduce

Render WebChat and view the page on High contrast mode on a Windows machine

## Expected behavior

Boundaries should be visible in high contrast mode

## Additional context

[Bug]

**Comment by corinagum** [comment]
Web Chat does not have control over the border of it's container. If you need to have high contrast borders, you will have to modify your app's Web Chat container styles.

I believe as a browser app, Web Chat is not responsible for ensuring High Contrast. Styling Web Chat to have a High Constrast mode would be the responsibility of the package users. To others experiencing this scenario, please note that `defaultStyleOptions` should contain the styles needed to enforce HC mode. If you run into options that are not available to modify that are needed to ensure an a11y pass, please feel free to file a new issue with that option specified.
